src/components/ha-expansion-panel.ts

```diff
@@ -0,0 +1,135 @@
+import {
+  css,
+  CSSResult,
+  customElement,
+  html,
+  LitElement,
+  property,
+  TemplateResult,
+} from "lit-element";
+
+@customElement("ha-expansion-panel")
+class HaExpansionPanel extends LitElement {
+  @property({ type: String }) title = "";
+
+  @property({ type: Boolean }) expanded = false;
+
+  protected render(): TemplateResult {
+    return html`
+      <div
+        class="summary ${this.expanded ? "expanded" : ""}"
+        @click=${this._togglePanel}
+      >
+        <div class="summary-content">
+          <p class="summaty-title">
```

Comment:
```suggestion
          <p class="summary-title">
```

Reply:
Also, lets not use p here since we are overwriting it anyway. Just use span

Reply:
fixed